script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_8B.txt

```diff
@@ -68,7 +68,7 @@ Arcueid stares deeply into my face,# and lets out a smile as if she had discover
 <Page19>
 ...She has completely seen through my thoughts.
 Arcueid's smile is so broad that it makes me somewhat uncomfortable.
-I don't like the feel of this―――but despite that, I can't help but wonder how many people this girl has killed until now.
+I don't like the feel of this―――but despite that, I can't help but wonder how many people this woman has killed until now.
```

Comment:
s/until now/before

Reply:
Before kinda reads weird to me. Might be in favour of just ending it at "how many people this woman has killed."

Reply:
works for me.